modules/hal_infineon/wifi-host-driver/CMakeLists.txt

```diff
@@ -49,7 +49,11 @@ zephyr_library_sources(${hal_wifi_dir}/WiFi_Host_Driver/src/whd_wifi_p2p.c)
 # src/bus_protocols
 zephyr_library_sources(${hal_wifi_dir}/WiFi_Host_Driver/src/bus_protocols/whd_bus.c)
 zephyr_library_sources(${hal_wifi_dir}/WiFi_Host_Driver/src/bus_protocols/whd_bus_common.c)
-zephyr_library_sources(${hal_wifi_dir}/WiFi_Host_Driver/src/bus_protocols/whd_bus_sdio_protocol.c)
+if(CONFIG_AIROC_WIFI_BUS_SPI)
```

Comment:
It is better to use `zephyr_library_sources_ifdef`.

Reply:
Done.